test/src/test/java/org/corfudb/runtime/clients/LogUnitHandlerTest.java

```diff
@@ -118,6 +118,17 @@ public void canReadWrite()
                 .isEqualTo(testString);
     }
 
+    @Test
+    public void canGetTrimMarkAndLogSize()
+            throws Exception {
+        byte[] testString = "hello world".getBytes();
+        client.write(0, null, testString, Collections.emptyMap()).get();
+        LogData r = client.read(0).get().getAddresses().get(0L);
+        assertThat (client.getTrimMark().get()==0);
+        assertThat(client.getQuotaUsed().get() > testString.length);
+        assertThat (client.getSegmentSize(client.getTrimMark().get(), client.getLogTail().getNumberOfDependents()).get() > testString.length);
+    }
+
```

Comment:
To make sure we have code coverage for the case of multiple segments, it would be nice to add a test in which the cluster is segmented and we query for the log size to one specific log unit.